doctrine/associations.rst

```diff
@@ -365,7 +365,7 @@ Of course, if you know up front that you'll need to access both objects, you
 can avoid the second query by issuing a join in the original query. Add the
 following method to the ``ProductRepository`` class::
 
-    // src/AppBundle/Entity/ProductRepository.php
+    // src/AppBundle/Repository/ProductRepository.php
```

Comment:
I think to keep entities and repositories separately is a better option. Let's use the `Repository` namespace like in best practices. I'll push more changes